src/lib/Execute.sol

```diff
@@ -16,7 +16,7 @@ abstract contract Execute {
     function executeCall(address to, uint256 value, bytes calldata data) public returns (bytes memory executeData) {
         _checkCanExecuteCall();
         (address guard, bytes memory checkBeforeData) = _beforeExecuteCall(to, value, data);
-        executeData = Address.functionCallWithValue(to, data, value); // library checks for contract existence
+        executeData = _call(to, value, data);
```

Comment:
OZ's calls require that the called address is a smart contract. This is too restrictive as it limits cases like sending ETH to an EOA